cli/cli/helpers/grafloki/docker_grafloki.go

```diff
@@ -67,7 +71,17 @@ func createGrafanaAndLokiContainers(ctx context.Context, dockerManager *docker_m
 		return "", stacktrace.Propagate(err, "An error occurred getting Docker network id by Name: %v", bridgeNetworkName)
 	}
 
-	lokiArgs := docker_manager.NewCreateAndStartContainerArgsBuilder(lokiImage, LokiContainerName, bridgeNetworkId).
+	lokiImage := defaultLokiImage
+	if graflokConfig.LokiImage != "" {
+		lokiImage = graflokConfig.LokiImage
+	}
+
+	lokiUuid, err := uuid_generator.GenerateUUIDString()
+	if err != nil {
+		return "", stacktrace.Propagate(err, "An error occurred generating a uuid for Loki.")
+	}
+	lokiContainerName := fmt.Sprintf("%v-%v", LokiContainerNamePrefix, lokiUuid)
```

Comment:
Same message above - I'd expect the prefix to be the full prefix, no `-` being included here

Reply:
agreed